benchmark/detect-libc.js

```diff
@@ -0,0 +1,39 @@
+const Benchmark = require('benchmark');
```

Comment:
Is the purpose of adding a benchmark test to verify that caching is occurring? I would imagine all these tests will be very, very fast on the 2nd and subsequent runs, and the timing of the 1st run will be lost to this.

Reply:
Yep, this is just to ensure the caching is working.

I almost introduce a flag to disable the cache just to be able to benchmark it.

And all these tests are fast, although the asynchronous version is not as fast as the sync version.